coop_shift/data/coop_shift_data.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="config_report_url" model="ir.config_parameter">
```

Comment:
Can you please remove this `config_report_url` record?
`report.url` parameter depends on the environment, it shouldn't be in this module.